certified-connectors/Impexium/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1850 @@
+                        "items": {},
+                        "description": "categories"
+                      },
+                      "customFields": {
+                        "type": "array",
+                        "items": {},
+                        "description": "customFields"
+                      },
+                      "showInDirectory": {
+                        "type": "boolean",
+                        "description": "showInDirectory"
+                      },
+                      "links": {
+                        "type": "array",
+                        "items": {},
+                        "description": "links"
+                      },
+                      "oldId": {
+                        "type": "string",
+                        "description": "oldId"
+                      },
+                      "isDeceased": {
+                        "type": "boolean",
+                        "description": "isDeceased"
+                      }
+                    }
+                  },
+                  "description": "dataList"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Find Individual by ID or Email or RecordNumber",
+        "description": "Find Individual by ID or Email",
+        "operationId": "Find-Individual-IDOrEmail",
+        "parameters": [
+          {
+            "name": "idOrRecordNumberOrEmail",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "ID Or Record Number Or Email",
+            "x-ms-summary": "ID Or Record Number Or Email"
+          },
+          {
+            "name": "pageNumber",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "default": "1",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "IncludeDetails",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "default": "True",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "AppToken",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "69a489d9-53fd-43aa-b04c-5c15ed4c73dd",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "UserToken",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "98ae7807-0902-4e66-9b0a-bfe0ecd204de",
+            "x-ms-visibility": "internal"
+          }
+        ]
+      }
+    },
+    "/api/v1/Committees/{code}/Members": {
+      "post": {
+        "responses": {
+          "default": {
```

Comment:
@RjImpx Please 'replace' the "default" responses with expected status codes, otherwise users may see errors about incorrect schema instead of real error responses from your service in OpenAPI mode.